pages/contribute/rule-feedback.md

```diff
@@ -23,27 +23,27 @@ Start giving feedback on issues:
 
 ## Feedback and reviews for draft rules (Pull Requests on GitHub)
 
-Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
+Pull requests are rule drafts that are ready for peer review. You can assign yourself as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to review it. However, even if three people are already reviewing it, we can always use more eyes on things.
```

Comment:
"assign yourself as a reviewer" is incorrect in Github lingo.
"assign" is used by Github for the person(s) in charge of the PR, reviewers are "requested". There has been several occurrences of people "assigning" other rather than "requesting review" from them. I'm a bit afraid this may drive non Github-savy people to click the "assign" button rather than the "request review" one. Especially, there is a Github button called "assign yourself" which is **not** what we want people to click in order to be reviewers…

<img width="290" alt="assign-review" src="https://user-images.githubusercontent.com/19969498/103877088-e201f500-50d4-11eb-94e7-7443af69ad4d.png">


OTOH, I agree that "you can request a review from yourself" sounds plain wrong in regular language… 🤔 I'm not having a good suggestion here.

Reply:
changed "assign yourself..." to "add yourself...", which should now be hopefully less confusing.